source/shared/cpp/ObjectModel/ActionParserRegistration.h

```diff
@@ -16,19 +17,17 @@ namespace AdaptiveSharedNamespace {
         virtual std::shared_ptr<BaseActionElement> Deserialize(
             std::shared_ptr<AdaptiveSharedNamespace::ElementParserRegistration> elementParserRegistration,
             std::shared_ptr<AdaptiveSharedNamespace::ActionParserRegistration> actionParserRegistration,
-            std::vector<std::shared_ptr<AdaptiveCardParseWarning>>& warnings,
-            const Json::Value& value) = 0;
+            std::vector<std::shared_ptr<AdaptiveCardParseWarning>>& warnings, const Json::Value& value) = 0;
```

Comment:
>, const Json::Value& value [](start = 76, length = 26)

We talked offline, but for posterity: I don't like that this has some of the parameters on their own lines and some that share a line. My preference would be that if they all fit on one line the all go on one line, but if not that each gets it's own line. Glancing at this i would likely miss seeing this value param and read it as having 3 parameters.

Reply:
I like this better :)

---
In reply to: [223460135](https://github.com/Microsoft/AdaptiveCards/pull/1996#discussion_r223460135) [](ancestors = 223460135)